Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 COPY --from=build-image ./app/dist ./dist
 COPY package* ./
 COPY migrations ./migrations
-RUN npm ci --production
+COPY migrateUpWithWrapper.mjs ./migrateUpWithWrapper.mjs
+RUN npm ci --omit=dev
```

Comment:
The new way of doing ` npm ci --production` is `npm ci --omit=dev`. It came to my attention from a warning logged when building the image.